### DISCUSSION TITLE
How to detect if script run in workspace?

### DISCUSSION THREAD

**Discussion by zfben** [community/community#128445] [npm]
Hi, I write an npm cli plugin that needs to read files from the node_modules folder.
When the plugin runs in a workspace, it found node_modules are empty.
Is there some way to detect if the plugin runs in a workspace and gets the real node_modules path?

**Comment by MylesBorins** [comment]
Hey @zfben, this seems like something we should document. The CLI team will work on documenting and update this feedback discussion once we have something for you to read.
